github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -161,6 +161,33 @@ presubmits:
           requests:
             memory: "8Gi"
 
+  - name: check-provision-k8s-1.20-cgroupsv2
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    max_concurrency: 1
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e@sha256:8c80c98d035a0f285ad49b964e5d0b62004844f3340407367841618e80e1503c
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - "cd cluster-provision/k8s/1.20 && CONTAINER_SUFFIX=cgroupsv2 CGROUPV2=1 ../provision.sh"
```

Comment:
Would it make sense to change `CGROUPV2` to `true` and for now also automatically in our scripting set the `CONTAINER_SUFFIX` to `cgroupsv2` if it `CGROUPV2` is set?

The less configuration knobs we expose here, the easier it is to change things.

Reply:
Yes, I can set `CONTAINER_SUFFIX` inside `provision.sh`. Just to clarify: are you suggesting to mkae `CGROUPV2=true` instead of `CGROUPV2=1`?

Reply:
Yes. Except if `1` has a different meaning than `true`.

Reply:
Ok, clear. Though CONTAINER_SUFFIX needs to match the suffix of the job's name here. Probably I can add that note as a comment in provision.sh then

Reply:
do you mean that they should match in the sense of a weak relationship, like identifying jobs which create this suffix?

Reply:
ah, no. I probably confused it with smth else. nevermind, it should be fine :)